plugins/serializers/registry.go

```diff
@@ -1,124 +1,131 @@
-package serializers
```

Comment:
Why does this file have so many "changes" when it appears to be the same file?

Reply:
LF vs CRLF newlines?